dev/com.ibm.ws.wssecurity_fat.wsscxf.saml.2/fat/src/com/ibm/ws/wssecurity/fat/cxf/samltoken2/common/CxfSAMLCallerTests.java

```diff
@@ -135,7 +135,7 @@ public void testCxfCallerHttpsPolicy() throws Exception {
     //1/26/2021 comment out
     //@Mode(TestMode.LITE)
     //scenario 3 - done
-    @Test
+    //@Test
```

Comment:
Should this be commented out so that it never runs or should it only not run with EE9, or some other setup?  We have rules that can be used to conditionally skip.